plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/commit/CommitViewImpl.java

```diff
@@ -180,7 +181,12 @@ public void setEnableCommitButton(boolean enable) {
     /** {@inheritDoc} */
     @Override
     public void focusInMessageField() {
-        message.setFocus(true);
+        new Timer() {
```

Comment:
Unfortunately it does not work in this place. It is a common practice to set such delay in our code.